addons/implied-repos/common/src/test/java/org/commonjava/indy/implrepo/fixture/TestValidReposStoreDataManager.java

```diff
@@ -0,0 +1,209 @@
+/**
```

Comment:
What does this class do? It looks like a decorator that only delegates and doesn't add anything...and I can't see it being used anywhere. Is this something old that didn't get deleted, or am I just missing something?